scripts/get_last_short_commit_hash.sh

```diff
@@ -10,11 +10,10 @@ TAG=""
 if ([ -z "$LATEST_COMMIT_TAG" ]); then
   # No commit tag found - staging deployment
   SHORTHASH="$(git rev-parse --short HEAD)"
-  TAG=$DOCKER_REPO:$SHORTHASH
+  TAG=$DOCKER_REPO$SHORTHASH
```

Comment:
why no `:`?

Reply:
they are built-in, the tag had "::" instead of ":" when I ran it like this